src/main/resources/db/migration/V1_12__service_user_data.sql

```diff
@@ -1,16 +1,18 @@
-create table service_user (
+alter table referral
+    add column service_usercrn text not null default 'unknown';
+
+create table referral_service_user_data (
     referral_id uuid not null unique,
+    nomis_number text,
+    pnc_number text,
     address text,
-    crn text,
```

Comment:
Does Address need to be in a separate table?

Reply:
i don't think so